app/dashboard/migrations/0110_auto_20200607_1455.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.4 on 2020-06-07 14:55
```

Comment:
I believe this should be git ignored?

Reply:
No, this is important because it track the changes in the models, so django can do the proper db changes to the schema to stay sync the db and the models